app/src/main/res/layout/dashboard_spotlight_concat.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data />
```

Comment:
if you aren't actually including any variables or imports, then there is no reason to include the `<data>` element.